src/components/NcNoteCard/NcNoteCard.vue

```diff
@@ -139,20 +151,27 @@ export default {
 
 <style lang="scss" scoped>
 .notecard {
+	--note-card-padding: calc(2 * var(--default-grid-baseline));
 	color: var(--color-main-text) !important;
 	background-color: var(--note-background) !important;
-	border-inline-start: 4px solid var(--note-theme);
+	border-inline-start: var(--default-grid-baseline) solid var(--note-theme);
 	border-radius: var(--border-radius);
-	margin: 1rem 0;
-	margin-top: 1rem;
-	padding: 1rem;
+	margin: 1lh 0;
```

Comment:
Isn't it making the margin 1.5 times larger?

Reply:
```suggestion
	margin: 1rem 0;
```